Validation/HcalRecHits/src/HcalRecHitsValidation.cc

```diff
@@ -378,11 +368,8 @@ void HcalRecHitsValidation::analyze(edm::Event const &ev, edm::EventSetup const
           else
             eHcalConeHFS += en;
         }
-        eHcalCone += en;
         nrechitsCone++;
 
-        HcalCone += en;
```

Comment:
This shouldn't have been removed. In fact, this is used to fill the histo below, now empty:
![image](https://user-images.githubusercontent.com/4069749/211153913-0c841f7b-53ad-4ec6-a935-c09cdf8ca184.png)

Reply:
ah my bad @perrotta , thanks for catching this. I have added this back